auth/auth.go

```diff
@@ -320,7 +320,7 @@ func (u *User) ID(ctx context.Context) *datastore.Key {
 	return UserID(ctx, u.Id)
 }
 
-func infoToUser(ui *oauth2service.Userinfoplus) *User {
+func infoToUser(ui *oauth2service.Userinfo) *User {
```

Comment:
I had to change this to get the service to start. I suspect it comes from some indirect dependency which was upgraded and the original type no longer exists?